speedy-recovery-client/src/components/core/App.js

```diff
@@ -284,120 +303,114 @@ class App extends Component {
   }
 
   updateStateCarePlan(userId) {
-      FhirDataQueryingService.getUserCarePlan(userId)
-          .then(carePlanResource => {
-              const carePlans = carePlanResource.map(carePlan =>
-                  fhirMapCarePlan(carePlan, this.state.fhirVersion)
-              );
-              this.setState({carePlans});
-          })
-          .catch(error => {
-              console.error(error);
-          });
+    FhirDataQueryingService.getUserCarePlan(userId)
+      .then(carePlanResource => {
+        const carePlans = carePlanResource.map(carePlan =>
+          fhirMapCarePlan(carePlan, this.state.fhirVersion)
+        );
+        this.setState({carePlans});
+      })
+      .catch(error => {
+        console.error(error);
+      });
   }
 
   updateStatePatient(patientResource) {
-      const filteredPatientResource = filterPatientResource(patientResource);
-      if (filteredPatientResource) {
-          const patient = fhirMapPatient(
-              filteredPatientResource,
-              this.state.fhirVersion
-          );
-          console.log("Patient Resource after mapping: ", patient);
-          this.setState({patient});
-      } else {
-          console.error(
-              "Crucial information missing from resource: ",
-              patientResource
-          );
-      }
+    const filteredPatientResource = filterPatientResource(patientResource);
+    if (filteredPatientResource) {
+      const patient = fhirMapPatient(
+        filteredPatientResource,
+        this.state.fhirVersion
+      );
+      console.log("Patient Resource after mapping: ", patient);
+      this.setState({ patient });
+    } else {
+      console.error(
+        "Crucial information missing from resource: ",
+        patientResource
+      );
+    }
   }
 
-  updateStateUser(currentUserResource) {
-      let user = undefined;
-      const filteredPatient = filterPatientResource(currentUserResource);
-      if (filteredPatient) {
-          user = fhirMapPatient(filteredPatient, this.state.fhirVersion);
-          console.log("User Resource after mapping: ", user);
-      } else {
-          console.error(
-              "Crucial information missing from resource: ",
-              filteredPatient
-          );
-      }
+  filterUser(currentUserResource) {
+    const filteredPatient = filterPatientResource(currentUserResource);
+    if (filteredPatient) {
+      const user = fhirMapPatient(filteredPatient, this.state.fhirVersion);
```

Comment:
Wouldn't this method rather be called `fhirMapUser`, and accordingly the `filterUserResource` and the corresponding methods in the Adapters? Because these run for all three types of user, right? Or maybe instead of user, we can use `Person`, so be more aligned with the FHIR terminology?